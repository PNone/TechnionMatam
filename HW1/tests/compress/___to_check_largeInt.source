baraa adi 95000000000 20:21
baraa adi 105000000000 20:20
baraa adi 95000000000 20:20
baraa adi 95000000000 20:20
baraa adi 105000000000 20:20
baraa adi 105000000000 20:19
baraa adi 95000000000 20:19
baraa adi 105000000000 20:10
baraa adi 95000000000 20:10
baraa adi 105000000000 20:10
baraa adi 95000000000 20:10
baraa adi 105000000000 20:10
baraa adi 95000000000 20:10
baraa adi 105000000000 20:10
baraa adi 95000000000 20:10