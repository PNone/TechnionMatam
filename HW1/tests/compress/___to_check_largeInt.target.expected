BlockChain Info:
1.
Sender Name: baraa
Receiver Name: adi
Transaction Value: 150
Transaction timestamp: 20:21
